dpx/Cargo.toml

```diff
@@ -10,15 +10,12 @@ license = "GPL"
 tectonic_bridge = { version = "0.0.1-dev", path = "../bridge" }
 libc = "0.2"
 libpng-sys = "1.1.8"
-libz-sys = { version = "1", optional = true}
 md-5 = "0.8.0"
 sha2 = "0.8.0"
 rand = "0.7.2"
 chrono = "0.4.9"
-
-[features]
-default = ['libz-sys']
-legacy-libz = ['libz-sys']
+# to ensure binaries are not different, we link to libz-sys
```

Comment:
If this is inaccurate, I'm not sure there's really a backwards-compatibility point in keeping the zlib linking. I think the default backend runs about as quickly as zlib.

Reply:
With zlib: https://tt.ente.ninja/#/compare/c188c51eadc8dfcc46b77819b7b8ac7d374e582b/5a5e070ef6bbdeb6e06d1956233d31e402a158cd

Reply:
With default, every file is different.